web/components/modals/ChildModal.tsx

```diff
@@ -73,8 +107,7 @@ function ChildModal({
                     <input
```

Comment:
Change the Date of Birth input to the component `/components/atoms/DatePicker/` to ensure that the date is valid. It's okay if the styles aren't aligned since we talked about updating the general inputs to components later